plugins/kubernetes/app/models/kubernetes/template_filler.rb

```diff
@@ -354,6 +357,7 @@ def set_secret_puller
           }
         }
       ]
+      volumes.uniq!
```

Comment:
unique per name?

Reply:
for now just dealing with exact copies, if 2 things have the same name and a different config, the user needs to see what is going on and figure out a compromise

Reply:
still pretty jank.. maybe just raise if they're the same?

Reply:
that's what we had before, having the same volume made samson blow up because the manifest became invalid
so users had to have a locally invalid manifest that uses a vaultauth mount but no volume